assignments/prototypes.js

```diff
@@ -35,41 +35,99 @@
   console.log(jumbo.isFlying)          // false
 */
 
-/*
 
-  TASK 1
 
-  - Build a Person Constructor that takes name and age.
-  - Give persons the ability to greet by returning a string stating name and age.
-  - Give persons the ability to eat edibles.
-  - When eating an edible, it should be pushed into a "stomach" property which is an array.
-  - Give persons the ability to poop.
-  - When pooping, the stomach should empty.
+  // TASK 1
 
-  TASK 2
+  // - Build a Person Constructor that takes name and age.
+  // - Give persons the ability to greet by returning a string stating name and age.
+  // - Give persons the ability to eat edibles.
+  // - When eating an edible, it should be pushed into a "stomach" property which is an array.
+  // - Give persons the ability to poop.
+  // - When pooping, the stomach should empty.
 
-  - Build a Car constructor that takes model name and make.
-  - Give cars the ability to drive a distance.
-  - By driving a car, the distance driven should be added to an "odometer" property.
-  - Give cars the ability to crash.
-  - A crashed car can't be driven any more. Attempts return a string "I crashed at x miles!", x being the miles in the odometer.
-  - Give cars the ability to be repaired.
-  - A repaired car can be driven again.
+  function Person (name, age){
+    this.name = name;
+    this.age = age;
+  
+  } ;
+  
+  Person.prototype.greet = function (){
+    return `Hello, I am ${this.name} and I am ${this.age} years old`
+  };
+  
+  Person.prototype.eatEdible = function (food){
+    stomach = [];
```

Comment:
You should nest `stomach` inside Person constructor, because stomach is a property of a person, right? You can access `stomach` property inside your `eatEdible` with keyword `this`. For example:
`return this.stomach.push(food);`